examples/nodejs-express-launchdarkly/lib/arcjet.js

```diff
@@ -0,0 +1,28 @@
+import _arcjet, { shield, slidingWindow } from "@arcjet/node";
+import { getArcjetConfig } from "./launchdarkly.js";
+
+// Initialize Arcjet with your site key and rules
+const aj = _arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://www.npmjs.com/package/dotenv
+  key: process.env.ARCJET_KEY,
+  rules: [],
+});
+
+// This function will return an Arcjet instance with the latest rules
+const arcjet = async () => {
+  // Get the latest configuration from LaunchDarkly
+  const newConfig = await getArcjetConfig();
+
+  // Return the Arcjet instance with the latest rules
+  return aj.withRule(shield({ mode: currentConfig.shieldMode })).withRule(
+    slidingWindow({
+      mode: currentConfig.slidingWindowMode,
+      max: currentConfig.slidingWindowMax,
+      interval: currentConfig.slidingWindowInterval,
+    })
+  );
+};
+
+export default arcjet;
```

Comment:
```suggestion
import arcjet, { shield, slidingWindow } from "@arcjet/node";
import { getArcjetConfig } from "./launchdarkly.js";

// Initialize Arcjet with your site key and rules
const aj = arcjet({
  // Get your site key from https://app.arcjet.com
  // and set it as an environment variable rather than hard coding.
  // See: https://www.npmjs.com/package/dotenv
  key: process.env.ARCJET_KEY,
  rules: [],
});

// This function will return an Arcjet instance with the latest rules
export default async function() {
  // Get the latest configuration from LaunchDarkly
  const config = await getArcjetConfig();

  // Return the Arcjet instance with the latest rules
  return aj.withRule(shield({ mode: config.shieldMode })).withRule(
    slidingWindow({
      mode: config.slidingWindowMode,
      max: config.slidingWindowMax,
      interval: config.slidingWindowInterval,
    })
  );
}
```
I started making these changes on your blog post but then I thought I should just do it here. Basically, if we are doing a default export, we don't need to name the function so we don't need a special prefix on `arcjet` and I think it looks nicer.